sdk/clients/tcgClient.go

```diff
@@ -0,0 +1,75 @@
+package clients
+
+import (
+	"context"
+	"github.com/rs/zerolog/log"
```

Comment:
`sdk` module should avoid 3rd party deps
"github.com/gwos/tcg/sdk/logper" should be used instead

Reply:
Moved the client out of the SDK for now.